File: solutions/javascript/line-up/1/line-up.js
//
// This is only a SKELETON file for the 'Line Up' exercise. It's been provided as a
// convenience to get you started writing code faster.
//

export const format = (customerName, customerNumber) => {
  let suffix='th';
  if(!([11,12,13].includes(customerNumber%100))){
    switch(customerNumber%10){
      case 1:
        suffix = 'st';
        break;
      case 2:
        suffix = 'nd';
        break;
      case 3:
        suffix = 'rd';
    }
  }
  return `${customerName}, you are the ${customerNumber}${suffix} customer we serve today. Thank you!`;
};

File: solutions/csharp/line-up/1/LineUp.cs
public static class LineUp
{
    public static string Format(string name, int number)
    {
        string suffix = "th";
        int[] weird_numbers = {11, 12, 13};
        if (!(weird_numbers.Contains(number%100))) suffix = (number%10) switch {
            1 => "st",
            2 => "nd",
            3 => "rd",
            _ => "th"
        };
        return $"{name}, you are the {number}{suffix} customer we serve today. Thank you!";
    }
}
